onedrive app is not a standard enterprise app (#205)
owncloud/docs-server: modules/admin_manual/pages/enterprise/external_storage/onedrive.adoc

== Introduction

> This is a consulting app, in order to get access to it you need to contact sales@owncloud.com first.
> Follow this guide to use Microsoft OneDrive as an external storage option in ownCloud.

Reviewer: ```suggestion
To use the ownCloud OneDrive app, you mandatory need consulting first. Contact sales@owncloud.com to get access. When agreed, follow this guide to use Microsoft OneDrive as an external storage option in ownCloud.
```
Improved the text a bit.
